### DISCUSSION TITLE
Error when i run the command npm run start

### DISCUSSION THREAD

**Discussion by luissotobedoya** [koltyakov/generator-sppp#84] [General]
Hello, a pleasure to greet you, I am trying to implement a project using the package and I get the following error:

![image](https://user-images.githubusercontent.com/42546975/216236929-a86b650f-6701-4f44-81dc-4d6962c19eaf.png)

What could it be?

i

**Comment by koltyakov** [comment]
Hi Luis,

Thanks for checking this.

Please bump `sp-build-tasks` dependency to v`3.19.4` in your project.

If you're running Node.js > 16, make sure to set up `NODE_OPTIONS=--openssl-legacy-provider` environment variable before running `npm run start` command.
